spec/controllers/orga/conferences_controller_spec.rb

```diff
@@ -25,6 +25,11 @@
         expect(response).to redirect_to (orga_conferences_path)
         expect(flash[:notice]).to match(/Import finished/)
       end
+
+      it 'refuses other formats' do
+        post :import, format: :json, params: { file: file}
```

Comment:
I _thought_ I added an extra check (on the Content_Type) in the POST request, after checking for .csv format in the form and before checking if the error is raised in the importer.rb. 
Now I am not sure if we need this in the first place? Or, I don't understand why to _not_ check the Content-Type here.